turnserver/internal/auth/auth.go

```diff
@@ -0,0 +1,65 @@
+package auth
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"os"
+	"path/filepath"
+	"sync"
+
+	"github.com/gravitl/netmaker/logger"
+	"github.com/gravitl/netmaker/turnserver/config"
+	"github.com/pion/turn/v2"
+)
+
+var (
+	authMapLock    = &sync.RWMutex{}
+	HostMap        = make(map[string]string)
+	authBackUpFile = "auth.json"
+	backUpFilePath = filepath.Join("/etc/config", authBackUpFile)
```

Comment:
missing slash?  /etc/config/
see next comment re: path

Reply:
filepath.join will add slash for comma separated values
yeah we could put it in /etc/turn/config for more isolation maybe